azurerm/resource_arm_application_insights.go

```diff
@@ -41,6 +41,7 @@ func resourceArmApplicationInsights() *schema.Resource {
 					"web",
 					"other",
 					"java",
+					"MobileCenter",
```

Comment:
Nitpick: I'm assuming that given the `DiffSuppressFunc` we can keep the casing consistent here? i.e. `mobilecenter`?

Reply:
we could - however I'm hoping we can remove the diffSuppressFunc here in future, so I'm going to leave this as-is for the moment.